docker/test/docker-compose.yml

```diff
@@ -1,69 +1,72 @@
-version: "3"
-
-networks:
-  thegraph:
-  blockchain:
-
 services:
   chain:
-    image: "iexechub/poco-chaintest:5.3.0-token-parity"
+    image: docker-regis.iex.ec/poco-chain:1.0.0-poco-v5.5.0-voucher-v1.0.0-nethermind
     restart: unless-stopped
-    networks:
-      - blockchain
     expose:
       - 8545
-      - 8546
+#      - 8546
     ports:
       - 8545:8545
-      - 8546:8546
+      # - 8546:8546 # port (not required for integration tests) fails to open on CI
 
   ipfs:
-    image: ipfs/go-ipfs:v0.10.0
     restart: unless-stopped
-    networks:
-      - thegraph
+    image: ipfs/go-ipfs:v0.22.0
+    expose:
+      - 8080
+      - 5001
```

Comment:
```suggestion
```
No need for expose if you use `ports` here.

Reply:
Sure see [Removed exposed ports which already mapped externally](https://github.com/iExecBlockchainComputing/PoCo-subgraph/pull/21/commits/47f83b650cfadfe9cb364f43e704de8ba377d1cc) :)